### DISCUSSION TITLE
System Requirements Specification (SRS) document

### DISCUSSION THREAD

**Discussion by TemirlanAidarov** [Automattic/wp-calypso#61462] [Q&A] [answered]
Hello!
I have just found about this project.
I am Computer Science student who takes a course related to Open Software Project.
We have an assignment to write/improve System Requirements Specification (SRS) document for any project. Basically, the SRS is a document that describes the nature of a project, software or application. In simple words, SRS document is a manual of a project provided it is prepared before you kick-start a project/application.

Can I write/improve SRS document for this project?

**Comment by lynnbgriffith** [comment, marked as answer]
Absolutely! You can find a lot of background information in the docs in the repo or at https://wpcalypso.wordpress.com/devdocs. If you have any questions, feel free to ping me in a discussion and I’ll at least get you pointed in the right direction.
